src/main/java/com/microsoft/graph/httpcore/FeatureFlag.java

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.graph.httpcore;
+
+public final class FeatureFlag {
+    private FeatureFlag(){}
+    public static final int NONE_FLAG = 0;
```

Comment:
enum instead?

Reply:
Chose this method because we want to store the specific values for these flags and these specific values need to increase by powers of 2 in order to keep the binary 'or' operator such that adding two flags will result in a binary solution where we only have to read the ones. If we did an enum their values would be in order from 0->(size-1). We could make the enum class and then add a separate class to extract values using a switch statement but that would make a call to get the feature flag value something like this -> featureFlag.NONE_FLAG.getVal() whereas right now we can just do featureFlag.NONE_FLAG.

Reply:
https://github.com/microsoftgraph/msgraph-sdk-java-core/pull/590/files#diff-6fcd223a874487abe1ec2c5a9e55276d39ad94fdc97c0adf273d2cfdf58b7c48

Reply:
Ah because we're sending that as hex representation to the service.  Disregard this comment them,  thanks